### PR TITLE
[APR] fix loading more and filter by pool types

### PR DESCRIPTION
**Pull request by luizakp**
This PR:
- Fix more pools load button
- remove pools that have bb-pools as tokens
- fix pool type filter

https://www.loom.com/share/bd1ac72dab39485fac0bbae122547c16?sid=383ed14e-c576-4a38-8f83-8ab214430a12

### CODE REVIEW

File: packages/gql/src/balancer/pools.ts [hunk 1]
```diff
@@ -195,12 +195,14 @@ export const aprPools = gql`
...
     $maxTvl: BigDecimal
     $block: Int
+    $tokensAsVunerabilityAffectedPools: [String!]
   ) {
     pools(
       where: {
         createTime_lte: $createdBefore
         totalLiquidity_gte: $minTvl
         totalLiquidity_lte: $maxTvl
+        tokens_: { address_not_in: $tokensAsVunerabilityAffectedPools }
```
Comment (ribeirojose) [inline packages/gql/src/balancer/pools.ts:205]: So we're not going to report this even historically?

**Review by ribeirojose — APPROVED** [review]
Lgtm, just left one question

File: packages/gql/src/balancer/pools.ts (hunk 1 above)
Comment (luizakp) [reply packages/gql/src/balancer/pools.ts:205]: 🫠 you are right

Comment (luizakp) [reply packages/gql/src/balancer/pools.ts:205]: added a tech debt here: https://linear.app/bleu-llc/issue/BAL-1165/handle-bb-pools-proportion-for-yield-apr 
talked to Fabio and the majority of bb pools have token yield, and the token apr will never be calculated without the price/proportion

So will filter out those pools and add them back into this task

**Pull request merged**